### PR TITLE
Translate Yotchang

### LINKED ISSUE #98: 英訳: 澁谷 典明 さんによる「cafebabepy(JVMで動くPython処理系)の構文解析もScalaにおまかせ！」
ref: #93

### PR DESCRIPTION
**Pull request by eed3si9n**
Fixes #98

### CODE REVIEW

File: _candidates_en/YoshiakiShibutani_1.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+organization: F-CODE, Inc.
+tags:
+  - Tools
+  - Others
+suggestions:
+  - Those interested in language implementations
+  - Those interested in Python
+  - Those considering adopting the parser combinator in Scala
+---
+I am currently making a Python implementation on JVM called cafebabepy, using Java. Still work in progress, really. For the REPL (read–eval–print loop), I've let Scala's parser combinator handle the syntactic analysis of cafebabepy.
```
Comment (smroy) [inline _candidates_en/YoshiakiShibutani_1.md]: Consider changing to "+I am currently making a Python implementation on JVM called cafebabepy, using Java. It's still a work in progress, really. For the REPL (read–eval–print loop), I've let Scala's parser combinator handle the syntactic analysis of cafebabepy."

File: _candidates_en/YoshiakiShibutani_1.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+In this talk, I will explain the overview of cafebabepy and practical syntactic analysis with the parser combinator library using the cafebabepy implementation as examples. Hopefully I can demonstrate how easy it is to use the parser combinators.
```
Comment (smroy) [inline _candidates_en/YoshiakiShibutani_1.md]: Consider changing to "+In this talk, I will explain the overview of cafebabepy and practical syntactic analysis with the parser combinator library using the cafebabepy implementation as an example. Hopefully I can demonstrate how easy it is to use the parser combinators."

Comment (eed3si9n) [reply _candidates_en/YoshiakiShibutani_1.md]: 👍

**Comment by eed3si9n** [comment]
Both suggestions sound good. Feel free to overwrite syntactic changes, assuming you can from the browser:

translate_yotchang_by_eed3si9n_ _pull_request__160_ _scalajp_2018_scalamatsuri_org

**Comment by smroy** [comment]
@eed3si9n I was unable to overwrite before, but my access to the repository has been adjusted and I can now edit without commenting and will review like this going forward. Thanks!

**Comment by eed3si9n** [comment]
Awesome.

**Pull request merged**